src/components/Link/Link.tsx

```diff
@@ -48,7 +48,9 @@ export interface LinkProps extends HTMLAttributes<HTMLAnchorElement> {
 const Link = ({ children, id, className, component: Component, variant, href, ...props }: LinkProps) => {
   const classNames = classnames(`${px}-link`, getLinkVariantClassName(variant ?? LinkVariants.standalone), className);
   const dataTestId = id ? `link-${id}` : `link`;
-  const isExternal = href.match(/https?/);
+  const isExternal = href.match(
+    /(http[s]?:\/\/)(?!.*phillips\.com)([a-zA-Z0-9\-.]+)(:[0-9]{1,4})?([a-zA-Z0-9/\-._~:?#[\]@!$&'()*+,;=]*)/g,
```

Comment:
can we add test cases  (as comments) what should we find using this RegEx? Would be great to have couple strings with valid cases and couple invalid